addon/components/frost-fixed-table.js

```diff
@@ -22,14 +22,17 @@ export default Component.extend({
   propTypes: {
     // options
     columns: PropTypes.arrayOf(ColumnPropType),
-    items: ItemsPropType
+    items: ItemsPropType,
+    onCallback: PropTypes.func
   },
 
   getDefaultProps () {
     return {
       // options
       columns: [],
-      items: []
+      items: [],
+      // do nothing by default, as the grid may not have any custom renderers that would need to emit events
+      onCallback: () => {}
```

Comment:
Why add a default function when your below action only makes the call when the function is present?